yacht/run_YACHT.py

```diff
@@ -65,7 +65,10 @@ def main(args):
 
     # Make sure the output can be written to
     if not os.access(outdir, os.W_OK):
-        raise FileNotFoundError(f"Cannot write to the location: {outdir}.")
+        # give error message, and exit with error status
+        print(f"Cannot write to the location: {outdir}.\n")
+        print("Please check if this location exists, and that you have the permission to write to this location. Exiting..\n")
+        sys.exit(1)
```

Comment:
I think the scope of this task was to add a test that raises such error, not changing yacht/run_YACHT.py

Reply:
https://github.com/KoslickiLab/YACHT/issues/44